staging/src/k8s.io/code-generator/cmd/validation-gen/validators/required.go

```diff
@@ -134,6 +179,65 @@ func (requirednessTagValidator) doOptional(context Context) (Validations, error)
 	return Validations{Functions: []FunctionGen{Function(optionalTagName, ShortCircuit|NonError, optionalValueValidator)}}, nil
 }
 
+// hasZeroDefault returns whether the field has a default value and whether
+// that default value is the zero value for the field's type.
+func (rtv requirednessTagValidator) hasZeroDefault(context Context) (bool, bool, error) {
+	t := realType(context.Type)
+	// This validator only applies to fields, so Member must be valid.
+	payload, hasDefault := gengo.ExtractCommentTags("+", context.Member.CommentLines)[defaultTagName]
```

Comment:
non-blocking, but I'll mention it: We've deprecated ExtractCommentTags

https://github.com/kubernetes/kubernetes/issues/130358 is opened to clean it up

Reply:
OK, will fix